Description: Change path of Perl to /usr/bin/perl for contrib/examples.
 Upstream uses /usr/local/bin/perl, but I like the contrib scripts to be 
 executable with just chmod +x on Debian systems.
Forwarded: not-needed
Author: Florian Hinzmann <fh@debian.org
Index: libnet-dns-perl/contrib/check_zone
===================================================================
--- libnet-dns-perl.orig/contrib/check_zone	2007-07-01 17:25:20.000000000 +0200
+++ libnet-dns-perl/contrib/check_zone	2010-03-15 10:28:02.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 # $Id: check_zone 638 2007-05-15 18:59:26Z olaf $
 
 =head1 NAME
Index: libnet-dns-perl/contrib/loc2earth.fcgi
===================================================================
--- libnet-dns-perl.orig/contrib/loc2earth.fcgi	2007-07-01 17:25:20.000000000 +0200
+++ libnet-dns-perl/contrib/loc2earth.fcgi	2010-03-15 10:28:02.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -T
+#!/usr/bin/perl -T
 
 # loc2earth.cgi - generates a redirect to Earth Viewer based on LOC record
 # [ see <URL: http://www.kei.com/homepages/ckd/dns-loc/ > or RFC 1876 ]
Index: libnet-dns-perl/demo/axfr
===================================================================
--- libnet-dns-perl.orig/demo/axfr	2007-07-01 17:25:20.000000000 +0200
+++ libnet-dns-perl/demo/axfr	2010-03-15 10:28:02.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 # $Id: axfr 264 2005-04-06 09:16:15Z olaf $
 
 use strict;
Index: libnet-dns-perl/demo/check_soa
===================================================================
--- libnet-dns-perl.orig/demo/check_soa	2007-07-01 17:25:20.000000000 +0200
+++ libnet-dns-perl/demo/check_soa	2010-03-15 10:28:02.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 # $Id: check_soa 264 2005-04-06 09:16:15Z olaf $
 
 =head1 NAME
Index: libnet-dns-perl/demo/check_zone
===================================================================
--- libnet-dns-perl.orig/demo/check_zone	2007-07-01 17:25:20.000000000 +0200
+++ libnet-dns-perl/demo/check_zone	2010-03-15 10:28:02.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 # $Id: check_zone 264 2005-04-06 09:16:15Z olaf $
 
 =head1 NAME
Index: libnet-dns-perl/demo/example_recurse.pl
===================================================================
--- libnet-dns-perl.orig/demo/example_recurse.pl	2007-07-01 17:25:20.000000000 +0200
+++ libnet-dns-perl/demo/example_recurse.pl	2010-03-15 10:28:02.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 # Example usage for Net::DNS::Resolver::Recurse
 # Performs recursion for a query.
Index: libnet-dns-perl/demo/mresolv
===================================================================
--- libnet-dns-perl.orig/demo/mresolv	2007-07-01 17:25:20.000000000 +0200
+++ libnet-dns-perl/demo/mresolv	2010-03-15 10:28:02.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 # $Id: mresolv 264 2005-04-06 09:16:15Z olaf $
 
 =head1 NAME
Index: libnet-dns-perl/demo/mx
===================================================================
--- libnet-dns-perl.orig/demo/mx	2007-07-01 17:25:20.000000000 +0200
+++ libnet-dns-perl/demo/mx	2010-03-15 10:28:02.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 # $Id: mx 264 2005-04-06 09:16:15Z olaf $
 
 =head1 NAME
Index: libnet-dns-perl/demo/perldig
===================================================================
--- libnet-dns-perl.orig/demo/perldig	2007-07-01 17:25:20.000000000 +0200
+++ libnet-dns-perl/demo/perldig	2010-03-15 10:28:02.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 # $Id: perldig 264 2005-04-06 09:16:15Z olaf $
 
 =head1 NAME
Index: libnet-dns-perl/demo/trace_dns.pl
===================================================================
--- libnet-dns-perl.orig/demo/trace_dns.pl	2007-07-01 17:25:20.000000000 +0200
+++ libnet-dns-perl/demo/trace_dns.pl	2010-03-15 10:28:02.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 use strict;
 use warnings;
